bin/check_versions

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+override_message="I have confirmed that no JSON version change is needed"
```

Comment:
I have no problem with this wording.

I think perhaps the only thing left is to document this check and the override process in the repository README? Any good reasons not to do this?